BabyKeyboardLock/EventEffectHandler.swift

```diff
@@ -71,7 +71,213 @@ class EventEffectHandler {
         "z": ["zebra", "zip", "zap", "zig", "zoo", "zen", "zero", "zone"]
     ]
     
+    // Translation dictionaries for supported languages
```

Comment:
Hi, thanks for your contribution. My original idea of the speak word mode:  when you press a key, it should speak a simple word that starts with the key char, so a -> apple, b -> bus. We can't just translate the words to another language, a new language should have his own list. For example in German, the word cat -> `Katze`  should be put in the k list,  for Japanese apple (pronounce as Ringo), it should be put in the r list.

What do u think?

Reply:
Huh, that makes sense! 
The idea in my mind for this feature addition was to allow baby to learn simultaneously a word and its translation (for bilingual parents)

Reply:
Actually, during testing I ran into an issue:
If keyboard is in differnet layout (cyrillic, for example) -> there is no words returned in the lookup -> it just pronounces the letter itself.

I think it'd be great to look up the words in a different language for that

Reply:
That would be the feature to change the Main language though
What I added here is an Additional language for the words

Reply:
P.s. I actually added a bunch of features, one of them - the app now shows the words on the screen. This probably also makes the translations feature clearer.. 
![image](https://github.com/user-attachments/assets/e5be3142-db65-45c3-abd5-12f0ae4bc8f6)

I decided not to submit all PRs at once - because I've developed them one on top of another, so the diffs would be cumulative. You can take a look here, though, if you want - and tell me if you want me to submit them here

https://github.com/calmmage/baby_keyboard/pulls?q=is%3Apr

Reply:
I like add new features to the app, especially the show spoken words.  As long as we keep the default UI as simple as possible